Maple2.Server.Game/Manager/Items/EquipManager.cs

```diff
@@ -193,6 +203,20 @@ public class EquipManager {
         }
     }
 
+    public bool EquipCosmetic(Item cosmetic, EquipSlot equipSlot) {
```

Comment:
Can we add some comments to this function. I assume you need to add this for Hair/Face because they are not equipped from your inventory, and the one being "unequipped" is discarded.